docs/DevelopmentPlan/DevelopmentPlan.tex

```diff
@@ -49,12 +51,36 @@ \section{Team Member Roles}
 
 \section{Workflow Plan}
 
+We will be using git as a version control software with GitHub as our remote repository. The root branch will be \code{master} and development branch based on \code{master} will be \code{dev}. The developer will branch off from \code{dev} to work on issues. There will be a new branch for each issue and will be named comtaining the issue number and name detailed below. Continuous Integration is set up between all branches \code{dev}, \code{master}, and Issues. Once the milestone is finished to the degree we are happy to be graded, the \code{dev} branch will be merged into \code{master}.
```

Comment:
- Spelling error on third sentence: There will be a new branch for each issue and will be named **containing** the issue number and name detailed below